thirdeye/thirdeye-frontend/app/pods/rootcause/route.js

```diff
@@ -231,6 +233,7 @@ export default Ember.Route.extend(AuthenticatedRouteMixin, {
     // session-initialized context
     if (sessionId) {
       if (!_.isEmpty(session)) {
+        const { name, text, updatedBy, updated } = model.session;
```

Comment:
I wonder if this is more confusing than helpful up here

Reply:
I think it's more concise than having to write `model.session.name`, `model.session.text`, etc.